package.json

```diff
@@ -251,7 +251,8 @@
     "prisma": "^5.21.1"
   },
   "overrides": {
-    "stream-shift": "1.0.2"
+    "stream-shift": "1.0.2",
+    "node-abi":"3.71.0"
```

Comment:
issue: Prebuildify has a dependency to node-abi.

> "node-abi": "^3.3.0"

It allows node-abi 3.71.0 by default.
We only need to update the sub dependencies of Prebuildify.

Reply:
I will remove it and only the contributing.md updates is required then.

Reply:
Could you please update the sub dependency in our tree?
The lock file pins it to 3.62.0 rn. Please update